api/src/main/java/org/cardanofoundation/rosetta/api/block/model/domain/Delegation.java

```diff
@@ -6,7 +6,7 @@
 import lombok.NoArgsConstructor;
 
 @Data
-@Builder //TODO saa: refactor tests and remove builder and *argConstructor annotations
```

Comment:
Do we remove these TODOs because it's not applicable anymore and the code is fixed?

Reply:
Yes, I decided to remove it, because our general approach to use Builders, I need only def and all args constructors, so it is not contradict the model class. After refactoring @Builders is not required but tests are already written with it...